site/content/entry/accurate-audio-emulation.md

```diff
@@ -0,0 +1,240 @@
++++
+date = "2018-12-07T08:00:00-08:00"
+title = "Accurate Audio Emulation Has Arrived"
+tags = [ "feature-update" ]
+author = "jroweboy"
+banner = "accurate-audio-emulation.png"
+forum = 67781
++++
+
+## Pokémon X / Y and Many More Games Are Finally Working!
+
+You've been asking for it for years now, and we've been listening, we promise!
+
+{{< figure src="/images/entry/accurate-audio-emulation/pokemonleague.png" 
+    title="The Pokémon league awaits your challenge!" 
+>}}
+
+<!--more-->
+
+Thanks to the hard work of one of our very talented developers, some of Citra's longest standing issues are finally fixed! Special thanks to all those who are supporting these efforts on [Patreon](https://www.patreon.com/citraemu). These donations are given directly to support the hard working developers such as [wwylele](https://github.com/wwylele/) who spent almost an entire year of his spare time on the feature in this blog post! We love working on this project, and have a whole lot more to talk about in the coming weeks!
+
+It's been a long time in development, but we are finally pleased to announce that many of the oldest bugs in Citra are now fixed in the latest Canary build thanks to the tireless efforts of [wwylele](https://github.com/wwylele/).
+Among the titles that had issues with Citra's HLE audio emulation, one stands out as the number one most requested game of all time: Pokémon X and Y.
+Before we get too much into the long story behind this great achievement, we should set expectations for what this means for the users of the emulator.
+
+## F.A.Q
+
+* What games are now working?
+
+We've been able to test a few of the fan favorites such as Pokémon X / Y, Fire Emblem Fates and Echoes, and many more! If you've experienced audio crashes or bugs in the past, now's the best time to try those games out again and help us find any issues with this new accurate audio feature.
+
+* How can I test it out?
+
+In the Audio tab of the Configuration menu, there is a new option for Emulation. Selecting "LLE - Accurate" will use the new feature, while the default value "HLE - Fast" will continue to use the original audio code.
```

Comment:
Isn't it "LLE (accurate)" ?

Reply:
yes. HLE (fast) and LLE (Accurate) My mistake on the first review.